src/python/ensembl/io/genomio/gff3/simplifier.py

```diff
@@ -71,7 +71,7 @@ class GFFSimplifier:
     # Multiple parameters to automate various fixes
     skip_unrecognized = True
     gene_cds_skip_others = False
-    allow_pseudogene_with_CDS = False
+    allow_pseudogene_with_cds = False
```

Comment:
I've added them explicitly to _init_, and I've added doc to explain what they are for.

Also yes, whatever the value of `skip_unrecognized`, I've added a `logging.warning()` to print the list of unknown biotypes (instead of only raising an exception).